frontend/packages/dev-console/src/components/monitoring/alerts/MonitoringAlerts.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import * as _ from 'lodash';
+// FIXME upgrading redux types is causing many errors at this time
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import { useSelector, useDispatch } from 'react-redux';
+import { match as RMatch } from 'react-router-dom';
+import { Table, TableHeader, TableBody, SortByDirection } from '@patternfly/react-table';
+import { FilterToolbar } from '@console/internal/components/filter-toolbar';
+import { getURLSearchParams } from '@console/internal/components/utils';
+import { getAlertsAndRules } from '@console/internal/components/monitoring/utils';
+import { monitoringSetRules } from '@console/internal/actions/ui';
+import { useURLPoll } from '@console/internal/components/utils/url-poll-hook';
+import { PrometheusRulesResponse } from '@console/internal/components/monitoring/types';
+import { RootState } from '@console/internal/redux';
+import { alertSeverityOrder, alertStateOrder } from '@console/internal/reducers/monitoring';
+import {
+  monitoringAlertRows,
+  monitoringAlertColumn,
+  alertRowFilters,
+  ruleStateOrder,
+} from './monitoring-alerts-utils';
+import './MonitoringAlerts.scss';
+
+type MonitoringAlertsProps = {
+  match: RMatch<{
+    ns?: string;
+  }>;
+};
+
+const POLL_DELAY = 15 * 1000;
+
+const MonitoringAlerts: React.FC<MonitoringAlertsProps> = ({ match }) => {
+  const {
+    'rowFilter-alert-state': alertStateParams,
+    'rowFilter-alert-severity': alertSeverityParams,
+    name,
+  } = getURLSearchParams();
+  const [sortBy, setSortBy] = React.useState({});
+  const [rows, setRows] = React.useState([]);
+  const dispatch = useDispatch();
+  const rules = useSelector((state: RootState) => state.UI.getIn(['monitoring', 'devRules']));
+  const namespace = match.params.ns;
+  const { prometheusTenancyBaseURL } = window.SERVER_FLAGS;
+  const [response, loadError, loading] = useURLPoll<PrometheusRulesResponse>(
+    `${prometheusTenancyBaseURL}/api/v1/rules?namespace=${namespace}`,
+    POLL_DELAY,
+    namespace,
+  );
+  const thanosRules = React.useMemo(
+    () => (!loading && !loadError ? getAlertsAndRules(response?.data).rules : []),
+    [response, loadError, loading],
+  );
+
+  React.useEffect(() => {
+    const sortThanosRules = _.sortBy(thanosRules, (rule) => ruleStateOrder(rule));
+    dispatch(monitoringSetRules('devRules', sortThanosRules));
+  }, [dispatch, thanosRules]);
+
+  const filteredRules = React.useMemo(() => {
+    if (alertStateParams || name || alertSeverityParams) {
+      return _.filter(
+        rules,
+        (r) =>
+          (name && _.includes(_.lowerCase(r.name), _.lowerCase(name))) ||
+          _.includes(alertStateParams, _.lowerCase(r?.state)) ||
```

Comment:
Shouldn't need to convert `state` to lower case (same as https://github.com/openshift/console/pull/5785/files#r452628730)